pkg/detectors/cloudflarecakey/cloudflarecakey.go

```diff
@@ -20,7 +20,8 @@ var _ detectors.Detector = (*Scanner)(nil)
 var (
 	client = common.SaneHttpClient()
 
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"cloudflare"}) + `\b(v[A-Za-z0-9._-]{173,})\b`)
+	// origin ca keys documentation: https://developers.cloudflare.com/fundamentals/api/get-started/ca-keys/
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"cloudflare"}) + `\b(v1\.0-[A-Za-z0-9-]{171})\b`)
```

Comment:
As per the [documentation](https://developers.cloudflare.com/fundamentals/api/get-started/ca-keys/) origin ca keys always start with `v1.0-`

Reply:
I generated around 5 keys and all of them had same length size of 171 excluding `v1.0-`

Reply:
I also thought to remove `cloudflare` as prefix because we have a fix `v1.0-` as prefix for each key. I would like input from others on this, if we do this or not.